patches/500_add_columns_reference_metadata_start_end_date.sql

```diff
@@ -0,0 +1,19 @@
+-- ****************************************************************************
+-- RM SQL DATABASE UPDATE SCRIPT
+-- ****************************************************************************
+-- Number: 500
+-- Purpose: Add reference, metadata, start date and end date columns
+--          to collection_exercise table
+-- Author: Leo Howard
+-- ****************************************************************************
+
+ALTER TABLE casev3.collection_exercise
+    ADD COLUMN IF NOT EXISTS reference varchar(255) NOT NULL;
+
+ALTER TABLE casev3.collection_exercise
+    ADD COLUMN IF NOT EXISTS start_date timestamp with time zone NOT NULL;
```

Comment:
won't work because the DB will have rows in it, which will be null.

suggest you create the column nullable, then put in a load of dummy dates, then flip the column to be not null.

See previous patch (300) for inspiration